languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -25669,6 +25671,46 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <suggestion>amende</suggestion>
                 <example correction="amende">J’ai été mis à l’<marker>amande</marker>.</example>
             </rule>
+            <rule><!--rule5-->
+                <pattern>
+                    <marker>
+                        <token regexp="yes" skip="-1">amende|amendes</token>
+                    </marker>
+                    <token regexp="yes" inflected="yes">&context_AMANDE;</token>
+                </pattern>
+                <message>Le fruit 'amande' semble plus probable que la sanction 'amende'.</message>
+                <example correction="amande">Un gâteau aux <marker>amendes</marker>.</example>
+            </rule>
+            <rule><!--rule6-->
+                <pattern>
+                    <token regexp="yes" inflected="yes" skip="-1">&context_AMANDE;</token>
+                    <marker>
+                        <token regexp="yes">amende|amendes</token>
+                    </marker>
+                </pattern>
+                <message>Le fruit 'amande' semble plus probable que la sanction 'amende'.</message>
+                <example correction="amande">Un gâteau aux <marker>amendes</marker>.</example>
+            </rule>
+            <rule><!--rule7-->
+                <pattern>
+                    <marker>
+                        <token regexp="yes" skip="-1">amande|amandes</token>
+                    </marker>
+                    <token regexp="yes" inflected="yes">&context_AMENDE;</token>
+                </pattern>
+                <message>La sanction 'amende' semble plus probable que le fruit 'amande'.</message>
+                <example correction="amende">J’ai été condamné à une forte <marker>amande</marker>.</example>
+            </rule>
+            <rule><!--rule8-->
+                <pattern>
+                    <token regexp="yes" inflected="yes" skip="-1">&context_AMENDE;</token>
+                    <marker>
+                        <token regexp="yes">amande|amandes</token>
+                    </marker>
+                </pattern>
+                <message>La sanction 'amende' semble plus probable que le fruit 'amande'.</message>
+                <example correction="amende">J’ai été condamné à une forte <marker>amande</marker>.</example>.
```

Comment:
@LucieSteib There's an extra dot after `</example>` here as well on line 25712, try removing it and pushing it again—hopefully this time it works! 😄